webcompat/webhooks/helpers.py

```diff
@@ -30,7 +32,63 @@ def new_opened_issue(payload):
 
 def handle_type_media(payload):
     """Checks duplicity when webhook receive a type-media issue."""
-    pass
+    media_issues = get_media_issues(MEDIA_PATH)
+    known_media, issue = is_known_media(payload, media_issues)
+    # Wondering if I should specialize the different requests
+    # (or not) like in set_labels.
+    headers = {
+        'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN'])
+    }
+    if known_media:
+        # 1. Remove label status-needstriage
+        # DELETE /repos/:owner/:repo/issues/:number/labels/:name
+        # delete_labels(['status-needstriage'], issue['number'])
+        path = 'repos/{0}/{1}/labels/status-needstriage'.format(
+            app.config['ISSUES_REPO_URI'], issue['number'])
+        proxy_request('DELETE', path,
+                      headers=headers,
+                      data=json.dumps(payload))
+        # 2. Set status-duplicate label
+        set_labels(['status-duplicate'], issue['number'])
+        # 3. Add a comment on the issue pointing to the original one
+        # POST /repos/:owner/:repo/issues/:number/comments
+        comment = 'This is a duplicate of Issue #{}'.format(issue['duplicate'])
+        # add_comment(comment, issue['number'])
```

Comment:
So basically same reply than above about creating new methods. I was ready to create a new method specialized on adding comments.